pkg/storaged/logs-panel.jsx

```diff
@@ -34,8 +34,8 @@ export class StorageLogsPanel extends React.Component {
             "_SYSTEMD_UNIT=multipathd.service"
         ];
 
+        const search_options = { prio: "debug", _SYSTEMD_UNIT: "storaged.service,udisks2.service,dm-event.service,smartd.service,multipathd.service" };
         const url = "/system/logs/#/?prio=debug&_SYSTEMD_UNIT=storaged.service,udisks2.service,dm-event.service,smartd.service,multipathd.service";
-
-        return <LogsPanel title={_("Storage Logs")} match={match} max={10} goto_url={url} />;
+        return <LogsPanel title={_("Storage Logs")} match={match} max={10} search_options={search_options} goto_url={url} />;
```

Comment:
It looks rather not needed to pass both goto_url and search_options properties here.
I would keep the search_options object since it's more flexible and use it like this in the onClick handler.
```
cockpit.jump(cockpit.location.encode(['logs'], search_options))

```

Reply:
Ah thanks! I wanted to pass only one but did not find any easy way - totally missed `cockpit.location.encode` :) Updated